### PR TITLE
feat: add Tangle Testnet

### PR DESCRIPTION
**Pull request by shekohex**
This PR adds [Tangle Testnet](https://www.tangle.tools/) to the chainlist.

### CODE REVIEW

File: _data/chains/eip155-3799.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    "wss://testnet-rpc-archive.tangle.tools"
+  ],
+  "faucets": ["https://faucet.tangle.tools"],
+  "nativeCurrency": {
+    "name": "tTNT",
+    "symbol": "TTNT",
+    "decimals": 18
+  },
+  "infoURL": "https://docs.tangle.tools",
+  "shortName": "ttangle",
```
Comment (drewstone) [inline _data/chains/eip155-3799.json:18]: Maybe `tTangle`?

File: _data/chains/eip155-3799.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
+{
+  "name": "Tangle Testnet",
+  "chain": "TTNT",
```
Comment (drewstone) [inline _data/chains/eip155-3799.json:3]: Maybe `Tangle Testnet`?

File: _data/chains/eip155-3799.json [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  "rpc": [
+    "https://testnet-rpc.tangle.tools",
+    "https://testnet-rpc-archive.tangle.tools",
+    "wss://testnet-rpc.tangle.tools",
+    "wss://testnet-rpc-archive.tangle.tools"
+  ],
+  "faucets": ["https://faucet.tangle.tools"],
+  "nativeCurrency": {
+    "name": "tTNT",
+    "symbol": "TTNT",
```
Comment (drewstone) [inline _data/chains/eip155-3799.json:14]:
```suggestion
    "symbol": "tTNT",
```

File: _data/chains/eip155-3799.json (hunk 2 above)
Comment (drewstone) [inline _data/chains/eip155-3799.json:3]:
```suggestion
  "chain": "Tangle Testnet",
```

File: _data/chains/eip155-3799.json [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+    "wss://testnet-rpc-archive.tangle.tools"
+  ],
+  "faucets": ["https://faucet.tangle.tools"],
+  "nativeCurrency": {
+    "name": "Testnet Tangle Network Token",
+    "symbol": "tTNT",
+    "decimals": 18
+  },
+  "infoURL": "https://docs.tangle.tools",
+  "shortName": "ttangle",
```
Comment (drewstone) [inline _data/chains/eip155-3799.json:18]:
```suggestion
  "shortName": "tTangle",
```

**Review by ligi — APPROVED** [review]

**Pull request merged**